docs/cody/usage-and-pricing.mdx

```diff
@@ -102,7 +102,7 @@ The following table shows a high-level comparison of the three plans available o
 | **Embeddings (public code)**      | Supported                                                  | Supported                                                                                   | Supported                             |
 | **Keyword Context (local code)**  | Supported                                                  | Supported                                                                                   | Supported                             |
 | **Developer Limitations**         | 1 developer                                                | Up to 50 devs                                                                               | Scalable, consumption-based pricing   |
-| **LLM Support**                   | Anthropic + Starcoder for Chat, Commands, and Autocomplete | Choice of LLMs for Chat (VS Code only), Anthropic + Starcoder for Commands and Autocomplete | Bring Your Own LLM Key (experimental) |
+| **LLM Support**                   | Anthropic + Starcoder for Chat, Commands, and Autocomplete | Choice of LLMs for Chat (VS Code and JetBrains only), Anthropic + Starcoder for Commands and Autocomplete | Bring Your Own LLM Key (experimental) |
```

Comment:
```suggestion
| **LLM Support**                   | Anthropic + StarCoder for Chat, Commands, and Autocomplete | Choice of LLMs for Chat and Commands (VS Code and JetBrains only), StarCoder and Autocomplete | Bring Your Own LLM Key (experimental) |
```

- We don't officially to allow users to change the autocomplete model, it's going to be mostly StarCoder 
- Commands mostly use the model last used in chat and DO NOT use StarCoder
- Capitalization is StarCoder https://huggingface.co/bigcode/starcoder